kinto/__init__.py

```diff
@@ -14,6 +14,8 @@
 
 
 DEFAULT_SETTINGS = {
+    'cliquet.project_name': 'cloud storage',
+    'cliquet.project_docs': 'https://kinto.rtfd.org/',
     'cliquet.cache_backend': 'cliquet.cache.memory',
```

Comment:
Also this cannot be overridded yet, see: https://github.com/mozilla-services/kinto/issues/90